iModelJsNodeAddon/api_package/ts/src/test/DgnDb.test.ts

```diff
@@ -4,6 +4,7 @@
 *--------------------------------------------------------------------------------------------*/
 import { DbResult, Id64Array, Id64String, IModelStatus, OpenMode } from "@itwin/core-bentley";
 import { BlobRange, DbBlobRequest, DbBlobResponse, DbQueryRequest, DbQueryResponse, DbRequestKind, DbResponseStatus, ProfileOptions } from "@itwin/core-common";
+import { DomainOptions } from "@itwin/core-common/lib/cjs/BriefcaseTypes";

```

Comment:
is this meant to be imported from cjs like this? outside the barrel file?

Reply:
I did not realize it was imported like this. I think VSCode automatically did that when I used a type. I will fix it.